kafka-commons/src/test/java/uk/gov/homeoffice/digital/sas/kafka/consumer/KafkaConsumerServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.homeoffice.digital.sas.kafka.consumer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.homeoffice.digital.sas.Constants.TestConstants.KAFKA_INVALID_VERSION;
+import static uk.gov.homeoffice.digital.sas.Constants.TestConstants.KAFKA_JSON_MESSAGE;
+import static uk.gov.homeoffice.digital.sas.Constants.TestConstants.KAFKA_VALID_VERSION;
+import static uk.gov.homeoffice.digital.sas.kafka.constants.Constants.KAFKA_CONSUMING_MESSAGE;
+import static uk.gov.homeoffice.digital.sas.kafka.constants.Constants.KAFKA_SCHEMA_INVALID_VERSION;
+
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.system.CapturedOutput;
+import org.springframework.boot.test.system.OutputCaptureExtension;
+import uk.gov.homeoffice.digital.sas.config.TestConfig;
+import uk.gov.homeoffice.digital.sas.kafka.message.KafkaAction;
+import uk.gov.homeoffice.digital.sas.kafka.message.KafkaEventMessage;
+import uk.gov.homeoffice.digital.sas.kafka.validators.SchemaValidator;
+import uk.gov.homeoffice.digital.sas.model.Profile;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+@SpringBootTest(classes = TestConfig.class)
```

Comment:
nit: this `TestConfig` class has the `EnableJpaRepositories` annotation which scans for repos. But I don't think we need that for this test.
Very tiny impact, but maybe it's worth creating a separate test config which excludes that to avoid bloating test classes with unnecessary preparation.